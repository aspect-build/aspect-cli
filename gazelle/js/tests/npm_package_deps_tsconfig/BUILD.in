# gazelle:generation_mode update
# gazelle:js_tsconfig enabled
# gazelle:js_npm_package_target_name pkg
# gazelle:js_project_naming_convention tsc
# gazelle:js_npm_package enabled
