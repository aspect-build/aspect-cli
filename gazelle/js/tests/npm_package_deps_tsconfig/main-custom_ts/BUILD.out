load("@aspect_rules_js//npm:defs.bzl", "npm_package")
load("@aspect_rules_ts//ts:defs.bzl", "ts_config", "ts_project")
load("@npm//:defs.bzl", "npm_link_all_packages")

# gazelle:js_files m start.ts

npm_link_all_packages(name = "node_modules")

ts_project(
    name = "m",
    srcs = ["start.ts"],
    tsconfig = ":tsconfig",
)

npm_package(
    name = "pkg",
    srcs = [
        "package.json",
        ":m",
    ],
    visibility = ["//visibility:public"],
)

ts_config(
    name = "tsconfig",
    src = "tsconfig.json",
)
